frontend/src/components/ProgramActions.tsx

```diff
@@ -66,7 +67,7 @@ const ProgramActions: React.FC<ProgramActionsProps> = ({ status, setStatus }) =>
         data-testid="program-actions-button"
         type="button"
         onClick={() => setDropdownOpen((prev) => !prev)}
-        className="rounded px-4 py-2 hover:bg-gray-200 dark:hover:bg-gray-700"
+        className="rounded hover:bg-gray-200 dark:hover:bg-gray-700"
```

Comment:
@Rajgupta36 I believe the hover effect should be not on the button but on the icon itself. If you look closely, right now on hover the background of the icon changes color weirdly, making it look as if those three dots gets stretched.. or have a shadow or something.
I do believe adding the hover effect on the icon itself would work better here:
```
        <FontAwesomeIcon icon={faEllipsisV} className="text-gray-400 hover:text-gray-500 dark:hover:text-gray-200" />
```
Maybe something like this?